tests/sandbox/test_sandbox_application.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from pytest_check import check
+import requests
+import json
+
+
+from data import Actor
+from typing import Callable
+from utils import HttpMethod
+
+
+@pytest.mark.sandbox
+class TestSandboxApplication:
+    @pytest.mark.parametrize("actor", Actor)
+    def test_get_codesystem(
+        self,
+        load_json: Callable[[str], str],
+        send_rest_request: Callable[[HttpMethod, str, Actor], requests.Response],
+        actor: Actor,
+    ):
+        codeSystemType = "SPECIALTY"
+        response = send_rest_request(
+            HttpMethod.GET, f"FHIR/STU3/CodeSystem/{codeSystemType}", actor
+        )
+
+        with check:
+            assert response.status_code == 200, (
+                f"UNEXPECTED RESPONSE: "
+                f"Actual response status code = {response.status_code}"
+            )
+
+        with check:
+            expected_response = load_json(
+                "getCodeSystem/responses/SpecialtyCodeSystem.json"
+            )
+            actual_response = json.loads(response.content)
+            assert expected_response == actual_response, (
+                f"\nUNEXPECTED RESPONSE: \n"
```

Comment:
Same here. I typically see lint complain for an f string without arguments.
I don't know if the fact that it's concatenating the strings is why it doesn't complain.

Reply:
Have also updated this to no longer be a format string.